account_followup/models/res_config.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Copyright 2004-2010 Tiny SPRL (<http://tiny.be>)
+# Copyright 2007 Tecnativa - Vicent Cubells
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import api, models, _
+
+
+class AccountConfigSettings(models.TransientModel):
+    _name = 'account.config.settings'
+    _inherit = 'account.config.settings'
+
+    @api.multi
+    def open_followup_level_form(self):
+        res_ids = self.env['account_followup.followup'].search([])
```

Comment:
* This is returning a recordset, not ids, so name it `res`.
* Add the condition `('company_id', '=', self.env.user.company_id.id)`.
* As there's a constraint, this will only return one record.